src/Sulu/Component/Content/Compat/LocalizationFinder.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Compat;
+
+use Sulu\Component\Localization\Localization;
+use Sulu\Component\Webspace\Manager\WebspaceManagerInterface;
+
+/**
+ * Find best localization.
+ */
+class LocalizationFinder
```

Comment:
at least dan has removed all of them (: this functionality was included in the subscriber ... therefor all other stuff was removed with the document manager refactoring!